src/renderer/components/ui/Sidebar/SidebarHeader.tsx

```diff
@@ -78,29 +78,14 @@ export const SidebarHeader = ({
         )}
       </Grid>
       <Grid item>
-        <Tooltip
-          title={tooltipText}
-          className={classes.tooltip}
-          placement='bottom'
-        >
-          <IconButton
-            className={classes.iconButton}
-            edge='end'
-            onClick={action}
-          >
+        <Tooltip title={tooltipText} className={classes.tooltip} placement='bottom'>
+          <IconButton className={classes.iconButton} onClick={action} edge='end'>
             <PlusIconWithBorder color='white' />
           </IconButton>
         </Tooltip>
       </Grid>
     </Grid>
   )
 }
-SidebarHeader.propTypes = {
-  classes: PropTypes.object.isRequired,
-  title: PropTypes.string.isRequired,
-  tooltipText: PropTypes.string.isRequired,
-  actions: PropTypes.arrayOf(PropTypes.element),
```

Comment:
@kowalski This prop used to be present in test only. All the other components containing this one were using 'action' of type 'func' instead so I changed it. Is it ok?